content/blog/making-a-slack-bot-with-node-and-mongo/index.md

```diff
@@ -0,0 +1,623 @@
+---
+{
+	title: "Making a Slack Bot using NodeJS and MongoDB",
+	description: 'Have you ever wanted to run native Java and Kotlin code from your mobile game written in Unity? Well you can! This article outlines how to set that up!',
+	published: '2020-02-04T05:12:03.284Z',
+	authors: ['crutchcorn'],
+	tags: ['mongodb', 'node', 'slack'],
+	attached: [],
+	license: 'cc-by-nc-sa-4'
+}
+
+---
+
+Modern-day remote live communication has never been as efficient or fast as it is today. You're able to use services like Slack to join huge multi-channel communication workspaces for pleasure or buisness. These channels are often able to be super-powered by in-chat bots and applications that can inform you of new information from external services or even add new functionality to the chat. Luckily for us, Slack has put a lot of effort into making these extensions to Slack easy to write.
+
+One way they've eased the effort of their creation is by providing an SDK for Node developers to take and create extensions with. This post will outline how we can create a Slack bot to add functionality to chats.
+
+# Initial Signup {#signup-for-dev-account}
+
+To start, we'll need to [signup for a developer account and create an app to host our applicaiton logic using this link](https://api.slack.com/apps). This will allow us to create new Slack apps and bots to add into our workspace.
```

Comment:
```suggestion
To start, we'll need to [sign up for a developer account and create an app to host our application logic using this link](https://api.slack.com/apps). This will allow us to create new Slack apps and bots to add into our workspace.
```